README.md

```diff
@@ -28,7 +30,8 @@ To run, Tracee requires the following:
 You can get Tracee in any of the following ways:
 1. Download the binary from the GitHub Releases tab (`tracee.tar.gz`).
 2. Use the docker image from Docker Hub: `aquasec/tracee`. The image already includes the required dependencies but you will need to mount the kernel headers in (see below for example).
-3. Build from source, using `make build` (or via Docker using `make build-docker`).
+3. Build from source via docker, using `make build-docker`.
+4. Build from source, using `make build` (requires libelf-dev in addition to the above prerequisites).
```

Comment:
I think this will be a transitive dependency of clang anyway. I explicitly installed it in the dockerfile because I'm using 1) a minimal container 2) --no-install-recommends. this is also the reason why I install llvm there which we should not mention here.

Reply:
I got some gelf.h missing problems with clang until I installed libelf-dev - I don't think this is a transitive dependency like you say